CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [4.23.0]
+
+### Added
+- In order to speed up the import process running import tasks in parallel was added. The option can be enabled in  Bigcommerce > Settings > Product Sync. Enabling it will add an additional background task runs on listings, channels initialization, and product data fetching.
+- Added support for customer webhooks. The options can be enabled during onboarding process or in Bigcommerce > Settings > Product Sync
```

Comment:
Just some edits on the content here:

"- Added the ability to run import tasks in parallel, in order to speed up the import process. This option can be enabled in Bigcommerce > Settings > Product Sync. Enabling it will add an additional background task that runs on listings, channels initialization, and product data fetching.
- Added support for customer webhooks, which will enable real-time data syncing between BigCommerce and BC4WP for customer related data, such as new customers being created and updated login settings. The options can be enabled during onboarding process or in Bigcommerce > Settings > Product Sync"